src/modules/site-v2/templates/_scripts/submit-job.js

```diff
@@ -18,20 +18,32 @@ $.ajaxSetup({
   If defining with `as_form_data` = True, must include `ajax_setup` on the page (defined above).
 #}
 {% macro def_submit_job(tool_name, as_form_data=false, func_name='submit_job') %}
-function {{func_name}}(data, modal_id, new_tab=false) {
+function {{func_name}}(data, modal_id, new_tab=false, propagate_err=true) {
```

Comment:
prefer `propagate_error` for clarity over abbreviations.

Reply:
I appreciate the `c-macro-ish-ness` nature of `{{ func_name }}`. Is there a chance it might be null/undefined/use reserved keyword, etc?

Reply:
Fixed the first part in [b137d57](https://github.com/AndersenLab/CAENDR/pull/402/commits/b137d5709aff41d74bc8177241ca161ae2e3d02d)

Reply:
To the second point - as it turns out, none of the calling sites use the `func_name` argument to change the name.  I think I put this in so we'd have a way to manually avoid name collisions (rather than the parameter being filled in by a variable), but that ended up not being necessary...

(To clarify -- I was envisioning something like, the following, using a string:
`{{ def_submit_job( ..., func_name='submit_job_foobar' ) }}`

...rather than something using a variable:
`{{ def_submit_job( ..., func_name=foobar ) }}`

...but since we don't use either, I could also just remove the parameter)

Reply:
got it, thanks!